README.md

```diff
@@ -160,13 +164,16 @@ algorithm,” Intel Corporation, vol. 5,no. 1-10, p. 4, 2001.
 An example using the Lucas-Kanade algorithm to determine the optical flow.
 
 ```julia
-using Images, TestImages, StaticArrays, ImageTracking, ImageView, LinearAlgebra, CoordinateTransformations, Gtk.ShortNames
+using ImageTracking
+using Images, ImageView
+using StaticArrays
 
 #=Image Credit:  C. Liu. Beyond Pixels: Exploring New Representations and
 #Applications for Motion Analysis. Doctoral Thesis. Massachusetts Institute of
 #Technology. May 2009. =#
-img1 = load("demo/table1.jpg")
-img2 = load("demo/table2.jpg")
+demo = joinpath(dirname(pathof(ImageTracking)), "..", "demo")
```

Comment:
FYI, a simpler version is possible in Julia 1.5: `joinpath(pkgdir(ImageTracking), "demo")`